classes/user/Collector.php

```diff
@@ -246,6 +248,17 @@ public function includeReviewerData(bool $includeReviewerData = true): self
         return $this;
     }
 
+    /**
+     * Defines whether active submissions count should be included
+     */
+    public function includeActiveSubmissions(int $userGroupId): self
+    {
+        $this->includeActiveSubmissions = [
+            'user_group_id' => $userGroupId,
```

Comment:
Do we need assigning associated array if filtering only by 1 parameter?

Reply:
I have tried that, but oddly when passing it to the query, it doesn't retrieve the active submissions number correctly for some reason. so I had to keep it as it is.